packages/asset-service/src/generateAssetData/Utils.test.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseAsset } from '@shapeshiftoss/types'
+
+import { BTCMockedAsset, ETHMockedAsset } from '../service/AssetServiceTestData'
+import blacklist from './blacklist.json'
+import { filterBlacklistedAssets } from './utils'
+
+const blacklistedAssets: string[] = blacklist
+
+jest.mock('./blacklist.json', () => [ETHMockedAsset.tokens![0].caip19, BTCMockedAsset.caip19], {
+  virtual: true
+})
+
+const assetList: BaseAsset[] = [
+  Object.assign({}, ETHMockedAsset),
+  Object.assign({}, BTCMockedAsset)
```

Comment:
Needed to clone these objects, overwise the test script mutate it and we're not able to test the final caip19 item